stablehlo/dialect/Register.cpp

```diff
@@ -32,6 +33,7 @@ void registerAllDialects(mlir::DialectRegistry &registry) {
   registry.insert<mlir::sparse_tensor::SparseTensorDialect>();
   registry.insert<mlir::chlo::ChloDialect,
                   mlir::stablehlo::StablehloDialect,
+                  mlir::stablehlo::interpreter::InterpreterDialect,
```

Comment:
Given that we don't currently expect the interpreter dialect to be used outside of the reference implementation, let's not add it here, because all our users are calling this function? Perhaps we can add this dialect manually to stablehlo-opt and stablehlo-translate?